scripts/local-deploy-bootstrap.py

```diff
@@ -79,6 +79,32 @@ def binary_available(name):
     return False
 
 
+def setup_profile():
+    """Check ~/.profile for a .bashrc source, adding it if necessary"""
+    create_profile = None
+
+    for profile in ('~/.bash_profile', '~/.bash_login', '~/.profile'):
+        fn = os.path.expanduser(profile)
+        if not os.path.isfile(fn):
+            continue
+
+        with open(fn) as f:
+            for line in f:
+                if line.strip() in ('. "$HOME/.bashrc"', '. ~/.bashrc'):
```

Comment:
Time for a regex, I think.

![xkcd 208](https://imgs.xkcd.com/comics/regular_expressions.png)